tmt/hardware.py

```diff
@@ -1097,7 +1152,15 @@ def _parse_cpu(spec: Spec) -> BaseConstraint:
             'model',
             'family',
             'vendor',
-            'stepping'
+            'stepping',
+            )
+        )
+
+    group.constraints += _parse_float_constraints(
+        spec,
+        'cpu',
+        (
+            'frequency',
```

Comment:
Here we would definitely specify the default unit, `MHz`, `_parse_number_constraints()` would then propagate it to `NumberConstraint` class. Note that this is similar to `memory`, but `memory` uses `SizeConstraint` where we do have a default unit already predefined, always `bytes`. For more generic "number" constraint, we can't predefined one, so we need to specify it when parsing a constraint.